src/Resources/views/Shop/Menu/_searchForm.html.twig

```diff
@@ -1,5 +1,18 @@
-{% form_theme settings.form '@BitBagSyliusElasticsearchPlugin/Shop/Menu/_searchFormTheme.html.twig' %}
+{# {% form_theme settings.form '@BitBagSyliusElasticsearchPlugin/Shop/Menu/_searchFormTheme.html.twig' %}
 
 {{ form_start(settings.form, {'attr': {'class': 'ui loadable form', 'novalidate': 'novalidate'}}) }}
 {{ form_row(settings.form.box) }}
-{{ form_end(settings.form, {'render_rest': false}) }}
+{{ form_end(settings.form, {'render_rest': false}) }} #}
```

Comment:
need to make this symfony form work - uncomment that, and pass class and dataset from below in here